### PR TITLE
Update data-model-rollup.md

### PR DESCRIPTION
**Pull request by XKange**
“成为 Base 表”修正为“称为 Base 表”

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If these changes need document changes, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/getting-started/data-model-rollup.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ ROLLUP 在多维分析中是“上卷”的意思，即将数据按某种指定
 
 ### 基本概念
 
-在 Doris 中，我们将用户通过建表语句创建出来的表成为 Base 表（Base Table）。Base 表中保存着按用户建表语句指定的方式存储的基础数据。
+在 Doris 中，我们将用户通过建表语句创建出来的表称为 Base 表（Base Table）。Base 表中保存着按用户建表语句指定的方式存储的基础数据。
```
Comment (EmmyMiao87) [inline docs/zh-CN/getting-started/data-model-rollup.md:344]: Please update English documents also.

**Review by EmmyMiao87 — CHANGES_REQUESTED** [review]

**Comment by EmmyMiao87** [comment]
Are you still working in this PR? @XKange

**Comment by zbtzbtzbt** [comment]
这个文档修正了吧，pr可以关了

File: docs/zh-CN/getting-started/data-model-rollup.md [hunk 2]
```diff
@@ -341,7 +341,7 @@ ROLLUP 在多维分析中是“上卷”的意思，即将数据按某种指定
 
 ### 基本概念
 
-在 Doris 中，我们将用户通过建表语句创建出来的表成为 Base 表（Base Table）。Base 表中保存着按用户建表语句指定的方式存储的基础数据。
```
Comment (zbtzbtzbt) [inline docs/zh-CN/getting-started/data-model-rollup.md:344]: 这个文档修正了，pr可以关了 @XKange

**Comment by jackwener** [comment]
This problem already has been fixed. closed.

**Pull request closed without merging**